invisible-tech/njord
# Feat: Bootstrap Njord Microservice

## Test Plan

1. Deploy this branch to `staging` on Heroku.
2. Change `include` and `rootDir` to be the `client` folder.
3. Run `yarn build`
4. Run the following script locally on a file on your `ignore` folder:
```js
require('dotenv').config()

const { recordEvent } = require('../build')

recordEvent({
  name: 'test',
  metadata: {
    test: 'test',
  },
})
  .then(result => console.log(result))
  .catch(err => console.log(err))
```
4.1 Look at the database to see if your test event was saved on the DB.
